workspace/assets/css/dev/dev.less

```diff
@@ -10,6 +10,18 @@
 	outline: rgba(255,0,0,36%) 1px solid;
 	outline-offset: 8px;
 }
+
+.show-img-without-alt {
+	img {
+		border: 2px solid red;
+	}
```

Comment:
I would do `img:not([alt])` to avoid setting css on img, because it can create a conflict.

I would also use outline instead of border

Reply:
img:not([alt]) is not working...

Reply:
It is working in chrome.. no?

Reply:
yep working. my bad